extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -712,78 +712,85 @@ export class AmpA4A extends AMP.BaseElement {
   /**
    * Render a validated AMP creative directly in the parent page.
    * @param {!ArrayBuffer} bytes The creative, as raw bytes.
-   * @return {Promise<boolean>} Whether the creative was successfully
+   * @return {boolean} Whether the creative was successfully
```

Comment:
This should fix a bug as well.
In line 425, we have `return creative && this.maybeRenderAmpAd_(creative);`
but that would have always returned `true` since `maybeRenderAmpAd_` used to return a promise which is truthy.